features/users.feature

```diff
@@ -269,6 +269,334 @@ Feature: Users
+                },
+                {
+                  "active": true,
+                  "email": "email7@ons.gov.uk",
+                  "forename": "Mary",
+                  "groups": [],
+                  "id": "id_3",
+                  "lastname": "Martin",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                }
+              ]
+            }
+            """
+
+    @get-users-list
+    Scenario: GET /v1/users and checking the response status 200 with sort by id:asc
+        Given a user with forename "Adam", lastname "Adams", email "email5@ons.gov.uk", id "id_2" and password "Passw0rd!" exists in the database
+        And a user with forename "William", lastname "Williams", email "email9@ons.gov.uk", id "id_1" and password "Passw0rd!" exists in the database
+        And a user with forename "Mary", lastname "Martin", email "email7@ons.gov.uk", id "id_3" and password "Passw0rd!" exists in the database
+        And I am an admin user
+        When I GET "/v1/users?sort=id:asc"
+        Then I should receive the following JSON response with status "200":
+            """
+            {
+              "PaginationToken": "",
+              "count": 3,
+              "users": [
+                              {
+                  "active": true,
+                  "email": "email9@ons.gov.uk",
+                  "forename": "William",
+                  "groups": [],
+                  "id": "id_1",
+                  "lastname": "Williams",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                },
+                {
+                  "active": true,
+                  "email": "email5@ons.gov.uk",
+                  "forename": "Adam",
+                  "groups": [],
+                  "id": "id_2",
+                  "lastname": "Adams",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                },
+                {
+                  "active": true,
+                  "email": "email7@ons.gov.uk",
+                  "forename": "Mary",
+                  "groups": [],
+                  "id": "id_3",
+                  "lastname": "Martin",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                }
+              ]
+            }
+            """
+
+    @get-users-list
+    Scenario: GET /v1/users and checking the response status 200 with sort by id:desc
+        Given a user with forename "Adam", lastname "Adams", email "email5@ons.gov.uk", id "id_2" and password "Passw0rd!" exists in the database
+        And a user with forename "William", lastname "Williams", email "email9@ons.gov.uk", id "id_1" and password "Passw0rd!" exists in the database
+        And a user with forename "Mary", lastname "Martin", email "email7@ons.gov.uk", id "id_3" and password "Passw0rd!" exists in the database
+        And I am an admin user
+        When I GET "/v1/users?sort=id:asc"
```

Comment:
Think this should be `:desc` from the scenario description. 

Which means the json needs to be inverted.